src/contents/articles/cold-drinks-were-the-remedy.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Cold Drinks Were the Remedy
+author: msanatan
+date: 2015-06-29
+template: article.jade
+---
+
+What do Code and Drinks have in common? Not that much thankfully (imagine the bugs). Since we had an event for [coding](/articles/public-coding-roundup.html) recently, this time we got about the drinking part!
```

Comment:
What are you referring to when you say *'the drinking part'*? You start off by saying that Code and Drinks have nothing in common and right after you indicate that we need to fill some type of drinking quota.